README.md

```diff
@@ -118,6 +118,27 @@ sentry_set_commits(
 )
 ```
 
+#### Create deploy
+
+Creates a new release deployment for a project on Sentry.
+
+```ruby
+sentry_create_deploy(
+  api_key: '...', # Do not use if using auth_token
+  auth_token: '...', # Do not use if using api_key
+  org_slug: '...',
+  project_slug: '...',
+  version: '...',
+  app_identifier: '...', # pass in the bundle_identifer of your app
+  env: 'staging', # The environment for this deploy. Required.
+  name: '...', # Optional human readable name
+  deploy_url: '...', # Optional URL that points to the deployment
+  started: 1622630647, # Optional unix timestamp when the deployment started
+  finished: 1622630700, # Optional unix timestamp when the deployment finished
```

Comment:
should we mention its UTC?

Reply:
Unix timestamps are timezone independent AFAIK, don't think we need this?